command/server/server_command.go

```diff
@@ -143,8 +143,15 @@ func (c *ServerCommand) DefineFlags() {
 		FlagOptional: true,
 	}
 
-	c.FlagMap["max-peers"] = helper.FlagDescriptor{
-		Description: fmt.Sprintf("Sets the client's max peer count. Default: %d", helper.DefaultConfig().Network.MaxPeers),
+	c.FlagMap["max-inbound-peers"] = helper.FlagDescriptor{
+		Description: "Sets the client's maximum number of inbound peers allowded. Default: 40",
+		Arguments: []string{
+			"PEER_COUNT",
+		},
+		FlagOptional: true,
+	}
+	c.FlagMap["max-outbound-peers"] = helper.FlagDescriptor{
+		Description: "Sets the client's max no.of outbound peers allowded. Default: 10",
```

Comment:
Please try to be consistent with the descriptions.

In the description for `max-inbound-peers` you've used `maximum number of ...`, but here you abbreviate it to `max no.of`

Reply:
Sure, Commit [link](https://github.com/0xPolygon/polygon-edge/pull/343/commits/d23d22d3f927960379bf1336b3a8d197349c119d)